src/bsoncxx/document/view.cpp

```diff
@@ -27,6 +27,16 @@ namespace bsoncxx {
 BSONCXX_INLINE_NAMESPACE_BEGIN
 namespace document {
 
+namespace {
+bson_iter_t to_bson_iter_t(element e) {
```

Comment:
If you wanted, I think you could add this to bsoncxx/private/helpers and re-use it in both files and remove the dup definitions. Your call.